src/components/matomo/__specs__/Matomo.spec.js

```diff
@@ -0,0 +1,122 @@
+import { mount } from 'enzyme'
+import { createBrowserHistory } from 'history'
+import React from 'react'
+import { Router } from 'react-router'
+
+import MatomoContainer from '../MatomoContainer'
+
+describe('src | components | matomo | Matomo', () => {
+  let history
+
+  beforeEach(() => {
+    history = createBrowserHistory()
+    history.push('/router/path')
+  })
+
+  it('should push a new page displayed event', () => {
+    // given
+    const fakeMatomo = {
+      push: jest.fn(),
+    }
+    window._paq = fakeMatomo
```

Comment:
```suggestion
    const fakeMatomo = {
      push: jest.fn(),
    }
    window._paq = fakeMatomo
```

On peut remonter ça dans le beforeEach pour factoriser et éviter les multiples déclarations dans le // given